src/Components/MediaPresenter.ts

```diff
@@ -1,42 +1,58 @@
+import * as mobx from 'mobx';
+
 const MAXLEN = 10;
 
+
 export class MediaPresenter {
+  constructor() {
+    mobx.makeObservable(this);
+  }
+  // make need to change to .deep
+  @mobx.observable.deep
```

Comment:
Quiz question: why can't this be a `ref`?

Reply:
replied below :D